clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -159,17 +159,18 @@ private void handleMetalakeCommand() {
     FullName name = new FullName(line);
     String metalake = name.getMetalakeName();
     String outputFormat = line.getOptionValue(GravitinoOptions.OUTPUT);
+    ListOptions listOptions = new ListOptions(outputFormat);
```

Comment:
This doesn't currently add any value to the code and makes it harder to understand. ListOption is also a poor name as it is used in both list and details commands.

Reply:
displayOptions might be a better name? But as I previously commented we should only add this after we have more than one display option.

Reply:
So you mean we don't need a `ListOptions` wrapper at this stage because we currently have only one display option, namely outputFormat?

Reply:
I think so, it adds extra complexity and decreases code readability for little or no benefit. I think it would be useful if we had 2 or 3 options.